nbs/09_processors.ipynb

```diff
@@ -29,7 +29,7 @@
     "#|export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    print(_res.text[0])
</pre><p>Might be more expressive if NB getitem indexed into cells? Then you could do <code>print(_res[0].source)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/803/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Love this idea

Reply:
Unfortunately `__getitem__` seems to interfere with AttrDict  (posted a screenshot in Skype)